index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="description" content="Sample Microverse Hello Project">
+        <meta name="author" content="Albert Mensah-Ansah">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="style.css">
+        <title>My Portfolio</title>
+    </head>
+    <body>
+
+        <nav class="topnav">
+            <div class="welcome">Welcome</div>
+            <div>
+                <i class="fa fa-bars hamburgar"></i>
+            </div>
+        </nav>
+
+        <div class="container">
+            <div class="headline">
+                Hey There. I'm Albert
+                <div class="sub-headline">
+                    I'm a Software Developer
+                </div>
+            </div>
```

Comment:
Because this is your main title, it's wide better to use `h1` instead of `div`, it's more semantic.

Reply:
When I use h1 for the headline, the linter prevents me from adding a child <div>. I need to style the child heading separately from the main headline
@keroloussamy